contracts/external/interfaces/ISafe.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity ^0.8.18;
+
+/**
+ * @title ISafe Declares the functions that are called on a Safe by Safe protocol.
+ */
+interface ISafe {
```

Comment:
I put this in external because we have a separate repository for Safe and we would be only using the Safe interface. But makes sense to keep it in `interfaces/accounts`.